README.md

```diff
@@ -860,7 +860,7 @@ $server = new Server(array(
 
 Similarly, you can use the result of the `$next` middleware request handler
 function to modify the outgoing response.
-Note that as per the above documentation, the `$next` function may return a
+Note that as per the above documentation, the `$next` may return a
```

Comment:
"...the `$next` may return a..." reads a bit odd imho.

Reply:
May be "the `$next` variable" or "the callable `$next`"?

Reply:
In the other paragraph ↓, its called "`$next` middleware request handler", maybe that?

Reply:
Fixed.